src/Goldilocks/Program.cs

```diff
@@ -1,43 +1,29 @@
 using System.Text.Json.Serialization;
+using Goldilocks;
 
 var builder = WebApplication.CreateBuilder(args);
+builder.Logging.ClearProviders();
 
 builder.Services.ConfigureHttpJsonOptions(options =>
 {
-    options.SerializerOptions.AddContext<GoldilocksSerializer>();
+    options.SerializerOptions.AddContext<AppJsonSerializerContext>();
 });
 
 var app = builder.Build();
 
-var summaries = new[]
-{
-    "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
-};
+var sampleTodos = TodoGenerator.GenerateTodos().ToArray();
 
-app.MapGet("/weatherforecast", () =>
-{
-    var forecast = Enumerable.Range(1, 5).Select(index =>
-        new WeatherForecast
-        (
-            DateOnly.FromDateTime(DateTime.Now.AddDays(index)),
-            Random.Shared.Next(-20, 55),
-            summaries[Random.Shared.Next(summaries.Length)]
-        ))
-        .ToArray();
-    return forecast;
-});
+var todosApi = app.MapGroup("/todos");
+todosApi.MapGet("/", () => sampleTodos);
+todosApi.MapGet("/{id}", (int id) =>
```

Comment:
There is no benchmark for that right now, do you still keep it around?

Reply:
Happy to remove. @DamianEdwards  -  I know you wanted this to be as similar as possible to the template. In this case it won't impact RPS, in theory it could impact published size but probably not in this particular case. So it would be safe to delete.

Reply:
Then keep it, but maybe with a comment (it won't impact size ;)) just because in a week I will ask the same question :/

Same of the clearing of the logging provider, that it's deriving from the template but only in a way that it matches what benchmarks do.

Reply:
Keep it, with a comment for clarity. Technically it can impact startup time and per request performance (more routes in the tree).